assembl/static2/js/app/utils/timeline.js

```diff
@@ -163,4 +165,14 @@ export const getBarPercent = (_phase: Phase) => {
     }
   }
   return barWidth;
+};
+
+export const getPhaseId = (_timeline: Timeline, _identifier: string) => {
+  let timeline = _timeline;
+  const identifier = _identifier;
+  if (!timeline) {
+    timeline = [];
+  }
+  const phase = timeline.filter(p => p.identifier === identifier);
+  return phase[0]['@id'].split('/')[1];
```

Comment:
you should set a type for the return value of this function (I assume `string`), I think that there are missing checks here (`phase.length` can't be 0 ? do `phase[0]['@id'].split('/')` have at least 2 or more elements ?) and flow will help you with this